force-app/main/default/applications/MainlyDigital.app-meta.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomApplication xmlns="http://soap.sforce.com/2006/04/metadata">
+    <brand>
+        <headerColor>#0070D2</headerColor>
+        <shouldOverrideOrgTheme>false</shouldOverrideOrgTheme>
+    </brand>
+    <description>This app is designed to assist mainlydigital with creating templates for their new employees based on their individual roles, and track the progress of these employees as they carry out their assigned tasks.</description>
+    <formFactors>Small</formFactors>
+    <formFactors>Large</formFactors>
+    <isNavAutoTempTabsDisabled>false</isNavAutoTempTabsDisabled>
+    <isNavPersonalizationDisabled>false</isNavPersonalizationDisabled>
+    <isNavTabPersistenceDisabled>false</isNavTabPersistenceDisabled>
+    <label>MainlyDigital App</label>
+    <navType>Standard</navType>
+    <tabs>Template__c</tabs>
+    <tabs>Template_task__c</tabs>
```

Comment:
Is there any value to having tabs for the template and journey tasks? These aren't really useful outside of their associated journey and template. There's a limited number of tabs that can be displayed, so it's always a good idea to pick carefully.